licenses.txt

```diff
@@ -1,62 +1,419 @@
 
-Licenses
-========
+Managed manually
+----------------
 
-Backbone.js
-===========
-http://backbonejs.org/
+Library: backbone.js
+Site http://backbonejs.org/
 License: MIT
 
-
-Underscore
-==========
-http://underscorejs.org/
+Library: backbone-localstorage
+Site: https://github.com/jeromegn/Backbone.localStorage
```

Comment:
I have also stopped using localStorage.
Link to it is commented out in index.html
```<!-- <script src="{% static 'figure/3rdparty/backbone-localstorage.js' %}"></script> -->```